docs/commands.rst

```diff
@@ -280,9 +280,9 @@ priority levels ``6`` and ``7`` respectively.
 
 The modifier ``infix``, ``infix right`` and ``infix left`` can be used
 to specify whether the defined symbol is non-associative, associative to
-the right, or associative to the left. The priority levels are floating
-point numbers, hence a priority can (almost) always be inserted between
-two different levels.
+the right, or associative to the left.
+Priority levels are floating point numbers (positive or negative), hence a
```

Comment:
The grammar does not allow negative floats.

Reply:
Should we change the grammar to allow negative floats? 0.0 is allowed in Pratter 1.2.

Reply:
Why not indeed.

Reply:
I suggest we leave that for another pull request.

Reply:
Ok. Thanks to add an issue for the record.

Reply:
The doc still needs to be fixed: positive or negative -> non-negative.